modules/anti_nd/settings.hpp

```diff
@@ -8,6 +8,8 @@
 
 // If you want to disable a setting, set it to 0.
```

Comment:
This comment may now be misleading for the setting on line 11?

Reply:
agreed, moving it and adding another comment